internal/yarpctest/outboundtest/outbound_event.go

```diff
@@ -164,3 +171,25 @@ func assertEqualIfSet(t require.TestingT, want, got string, msgAndArgs ...interf
 		assert.Equal(t, want, got, msgAndArgs...)
 	}
 }
+
+type customRespBody struct {
```

Comment:
nit: At some point we should iron out resp vs res throughout. We are not consistent with our abbreviations. I favor Res because it’s pronounceable and same length as Req.